src/Missings.jl

```diff
@@ -87,6 +92,8 @@ Base.IteratorEltype(::Type{<:EachReplaceMissing{T}}) where {T} =
 Base.length(itr::EachReplaceMissing) = length(itr.x)
 Base.size(itr::EachReplaceMissing) = size(itr.x)
 Base.eltype(itr::EachReplaceMissing) = nonmissingtype(eltype(itr.x))
+Base.eachindex(itr::EachReplaceMissing) = eachindex(itr.x)
+Base.keys(itr::EachReplaceMissing) = keys(itr.x)
```

Comment:
These two will fail if `itr.x` does not define them, but maybe it is OK.

Reply:
Yes that's intended. We do the same for `SkipMissing` in Base.

Reply:
so maybe we should support `length`, `size` etc. (all the array interface) in the same way? We cannot do it for `SkipMissing` but here we can.

Reply:
We already do. :-)

Reply:
Ah, we need `axes`. Will add it.